### PR TITLE
Add vision support

### LINKED ISSUE #145: Expose multiple cameras
Currently the boardgroup is a protected attribute.

### LINKED ISSUE #146: Reconsider development status
It's currently Alpha, and i would definitely say that it's been beta since SP2019

### LINKED ISSUE #180: Remove casts in environment definitions
See https://github.com/srobo/sr-robot3/pull/2#discussion_r449811261 for equivalent in sr-robot3

Requires j5 >= 0.9.3

### LINKED ISSUE #183: Update to j5 0.13.0

### PR DESCRIPTION
**Pull request by tpoomlmly**
Adds support for Logitech C270 webcams. They're optional too.

Closes #145 
Closes #146 
Closes #180 
Closes #183

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,3 +43,18 @@ r = Robot(wait_start=False)
...
+You will need to print off a [chAruco marker grid](https://docs.opencv.org/4.5.3/charuco_board.png).
+
+`opencv_interactive-calibration -t=charuco -sz=GRID_SIZE`
+
+Replace GRID_SIZE with the length of one of the larger squares (in mm) from the printed marker grid.
+
+Use `-ci=1` for specifying camera index if multiple cameras are connected.
+
+Point the camera at the marker grid. Until DF is at or below 30 then press S to save.
+This will output a `cameraParameters.xml` file. Place this file in `sr/robot3/vision/calibrations` named by the camera model.
```
Comment (trickeydan) [inline README.md:58]: 👀

File: README.md [hunk 2]
```diff
@@ -43,3 +43,18 @@ r = Robot(wait_start=False)
...
+
+You will need to add a detection strategy for the camera in to `sr/robot3/vision/backend.py`.
```
Comment (trickeydan) [inline README.md:60]: 👀👀

**Review by trickeydan — COMMENTED** [review]
You may need to update the docs for the arduino after the j5 upgrade. I suggest we look at the changelog between 0.8.0 and 0.13.0 to see what changed

**Comment by tpoomlmly** [comment]
Tested it on a robot and it works

**Comment by trickeydan** [comment]
Was this tested with multiple cameras? I don't think `j5-zoloto` supports multiple yet

**Comment by tpoomlmly** [comment]
ah, no it wasn't

**Pull request merged**